about.html

```diff
@@ -90,20 +92,20 @@
     </nav>
     <main class="main-content">
       <section
-        class="intro container-fluid d-flex flex-column gap-4 p-md-5 p-2 p-lg-5 justify-content-center align-items-center"
+        class="introduce_main container-fluid"
       >
+      <div class="container d-flex flex-column gap-4 p-md-5 p-2 p-lg-5 justify-content-center align-items-center">
         <div
           class="container-fluid d-md-flex flex-md-column align-items-center gap-3"
         >
-          <h1 class="greetings">The Web Developer Bootcamp 2022</h1>
           <img
-            src="images/Blog-2-2.png"
+            src="images/about_title.png"
             class="w-100 d-md-none"
             alt="Bootcamp banner"
           />
           <img
-            src="images/Blog-2-2.png"
-            class="w-75 d-none d-md-block"
+            src="images/about_title.png"
+            class="w-50 d-none d-md-block"
             alt="Bootcamp banner"
           />
           <span class="line"></span>
```

Comment:
- [ ] great job removing the old image, but this title should be an HTML code styled with CSS and not an image, kindly try to remove the image and replace it with HTML Heading and try to style it like the template

![image](https://user-images.githubusercontent.com/89282221/190245676-adb378d4-4150-4159-9042-9a7c4daad604.png)